src/System.Drawing.Primitives/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1 @@
+user32.dll!GetSysColor
```

Comment:
@hughbe that's a question I'm not qualified to answer but considerations I assume include, is it physically available on all OneCore platforms,, is it using a technology that they want to deprecate, is it compatible with the app container sandboxing, does it work well with the UWP app model such as lifetime, settings, and sharing, etc. I think in general API related to HWNDs are not available because that isn't the UI model that apps use. It's all complex and evolving. We've asked for some API to be added as part of our ongoing UWP work and largely they've been able to add it. In some cases such as registry API it's not consistent with the UWP model and we either won't ship our API or we'll throw PNSE or do some kind of fallback.